### ISSUE TITLE
Assailer/Harbinger flavor text

### ISSUE THREAD

**Issue by TheOrgg** [buttonmen-dev/buttonmen#2894]
"The Assailer combines natural speed with devastating missile power.  When she's not casting death magic or raining missiles on her opponents from above, she enjoys beating people up."

"The Harbinger is an excellent light infantry unit with great melee and save potential.  Her hobbies include chariot riding, needlepoint, and beating people up."

**Comment by TheOrgg** [comment]
@blackshadowshade , is this a quick-and-easy?  I pulled out the pack I got from Rhea and saw the flavor text at the bottom.

**Comment by blackshadowshade** [comment]
Yes, it is, but I don't want it to conflict with the roujin pull, so I'm waiting for that to be resolved first.
